README.md

```diff
@@ -87,18 +87,18 @@ This gem will directly track the semantic versioning of the Twitter Bootstrap pr
 
 ## Contributing
 
-This gem is fully tested from Rails 3.1 to 4.1. We run our tests on [Travis CI](http://travis-ci.org/metaskills/less-rails-boostrap) in both Ruby 1.9, 2.0, 2.1, and jRuby 1.9 mode. If you detect a problem, open up a github issue or fork the repo and help out. After you fork or clone the repository, the following commands will get you up and running on the test suite.
+This gem is fully tested from Rails 4.0 to 5.2. We run our tests on [Travis CI](http://travis-ci.org/metaskills/less-rails-boostrap) in both Ruby 1.9, 2.0  and jRuby 9000. If you detect a problem, open up a github issue or fork the repo and help out. After you fork or clone the repository, the following commands will get you up and running on the test suite.
```

Comment:
Remove "We run our tests on [Travis CI](http://travis-ci.org/metaskills/less-rails-boostrap) in both Ruby 1.9, 2.0  and jRuby 9000." - That's true of the original project, but not our fork. We can add CI details if/when we hook this up to CircleCI.

Reply:
Is there any perspective to merge this? I am not really interested in this gem anymore but can apply your suggestion if it will unblock other people.